### ISSUE TITLE
Configure rubocop to ignore method length in tests

### ISSUE THREAD

**Issue by DeannaMarbeck** [wnbrb/wnb-rb-site#37]
Currently rubocop complains about the method length in spec files, so it needs to be disabled for that in each file.  It would be good if it was configured to always ignore method length in test files.

**Comment by wildmaples** [comment]
I don't think this issue is relevant anymore - running Rubocop doesn't present any issues.

Also, we have [`Metrics/MethodLength` disabled in our Rubocop configuration](https://github.com/wnbrb/wnb-rb-site/blob/main/.rubocop.yml#L13-L14).

**Comment by jrussell416** [comment]
Issue Review:

Recommendation: Close
Reason: Verified that running rubocop no longer presents an issue and checked the rubcop.yml that Metrics/MethodLength engabled is set to false as noted in above comment.
Related Pull Request: N/A

Next Steps: Close

**Comment by jrussell416** [comment]
Reviewing old issues to still if they are still relevant

This issue can be closed as there is no issue when running Robocop
